ui/partials/perf/comparectrl.html

```diff
@@ -10,8 +10,14 @@
     <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
       <h1>Perfherder Compare Revisions</h1>
       <ul class="list-inline push-information">
-        <li><strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
-        <li><strong>New</strong> - <a href="{{newProject.getRevisionHref(newRevision)}}">{{newRevision}}</a> ({{newProject.name}})</li>
+        <li>
+          <strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}})
+           - {{originalAuthor}} - {{originalComments}}
+        </li>
+        <li>
+          <strong>New</strong> - <a href="{{newProject.getRevisionHref(newRevision)}}">{{newRevision}}</a> ({{newProject.name}})
+          - {{newAuthor}} - {{newComments}}
+        </li>
```

Comment:
This change needs to be made in comparesubtestctrl.html as well. As I hinted at in the other bug, it would be good if we could turn this block into a directive so that we would only have to make these changes once in the future. See comments here:

https://github.com/mozilla/treeherder/pull/792#discussion_r35133692